assets/android/src/CompoundIcons.kt

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
```

Comment:
This file is new, should this read `2024`?

Reply:
I'll re-generate the tokens so all generated files will have an updated header.